src/test_l1_tvSettings.c

```diff
@@ -15954,32 +15956,28 @@ void test_l1_tvSettings_negative_GetTVGammaTarget (void)
     if (extendedEnumsSupported == true)
     {
         /* Step 01: Calling tvsettings GetTVGammaTarget before TvInit */
-        GetTVGammaTarget(colorTempValue, &x_Value, &y_Value);
-        UT_ASSERT_FALSE((x_Value >= 0 && x_Value <= 1.0));
+        result = GetTVGammaTarget(colorTempValue, &x_Value, &y_Value);
+        UT_ASSERT_EQUAL_FATAL(result, tvERROR_INVALID_STATE);
     }
     /* Step 02: Calling tvsettings initialization and expecting the API to return success */
     result = TvInit();
     UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
 
     /* Step 03: Calling tvsettings GetTVGammaTarget with invalid input*/
-    GetTVGammaTarget( tvColorTemp_MAX , &x_Value, &y_Value);
-    UT_ASSERT_FALSE((x_Value >= 0 && x_Value <= 1.0));
-    UT_ASSERT_FALSE((y_Value >= 0 && y_Value <= 1.0));
+    result = GetTVGammaTarget( tvColorTemp_MAX , &x_Value, &y_Value);
+    UT_ASSERT_EQUAL_FATAL(result, tvERROR_INVALID_PARAM);
 
     /* Step 04: Calling tvsettings GetTVGammaTarget with invalid input*/
-    GetTVGammaTarget( (tvColorTemp_t)-1 , &x_Value, &y_Value);
-    UT_ASSERT_FALSE((x_Value >= 0 && x_Value <= 1.0));
-    UT_ASSERT_FALSE((y_Value >= 0 && y_Value <= 1.0));
+    result = GetTVGammaTarget( (tvColorTemp_t)-1 , &x_Value, &y_Value);
+    UT_ASSERT_EQUAL_FATAL(result, tvERROR_INVALID_PARAM);
 
     /* Step 05: Calling tvsettings GetTVGammaTarget with invalid input*/
-    GetTVGammaTarget( colorTempValue, NULL, &y_Value);
-    UT_ASSERT_FALSE((x_Value >= 0 && x_Value <= 1.0));
-    UT_ASSERT_FALSE((y_Value >= 0 && y_Value <= 1.0 ));
+    result = GetTVGammaTarget( colorTempValue, NULL, &y_Value);
+    UT_ASSERT_EQUAL_FATAL(result, tvERROR_INVALID_PARAM);
```

Comment:
Dont use UT_ASSERT_EQUAL_FATAL instead use UT_ASSERT_EQUAL

Reply:
addressed in commit id:- 45ba5edef84a528bb3ef3e891c40bc01d1646c0a